storm-server/src/main/java/org/apache/storm/daemon/supervisor/Container.java

```diff
@@ -130,6 +131,10 @@ protected Container(ContainerType type, Map<String, Object> conf, String supervi
         this.ops = ops;
         this.conf = conf;
         this.supervisorId = supervisorId;
+        String numaId = SupervisorUtils.getNumaIdForPort(port, conf);
+        if (numaId != null) {
+            this.supervisorId +=  Constants.NUMA_ID_SEPARATOR + numaId;
```

Comment:
This should be removed according to the change in BasicContainer. mkLaunchCommand